app/models/email_template.rb

```diff
@@ -42,6 +40,10 @@ def users
       User.mentee.where.not(registration_token: nil)
     when :unregistered_mentors
       User.mentor.where.not(registration_token: nil)
+    when :missing_mentors
+      User.mentor.where(is_missing: true).where(missing_since: ((Time.now - 7.days)..Time.now))
+    when :missing_mentees
+      User.mentee.where(is_missing: true).where(missing_since: ((Time.now - 7.days)..Time.now))
```

Comment:
Hi @DMips .. the missing mails will be sent every week ? Or the time when someone reported his/her partner missing ? 
the query will depend accordingly. Right now it send to mail to users missing in last 7days